pkg/gotohelm/helmette/sprig.go

```diff
@@ -400,3 +402,9 @@ func Join[T any](sep string, s []T) string {
 	}
 	return out
 }
+
+// +gotohelm:builtin=sha256sum
+func Sha256Sum(input string) string {
+	hash := sha256.Sum256([]byte(input))
+	return hex.EncodeToString(hash[:])
```

Comment:
Remind me the purpose of `[:]`? I was under the impression that it's functionally a no-op or there's something tricky about strings vs slices vs arrays.

Reply:
Sum256 returns a fixed-length array; EncodeToSting wants a slice.